lib/jmap/mail/email/email.dart

```diff
@@ -0,0 +1,137 @@
+import 'package:equatable/equatable.dart';
+import 'package:jmap_dart_client/http/converter/email_id_nullable_converter.dart';
+import 'package:jmap_dart_client/http/converter/email_id_converter.dart';
+import 'package:jmap_dart_client/http/converter/keyword_identifier_converter.dart';
+import 'package:jmap_dart_client/http/converter/mailbox_ids_converter.dart';
+import 'package:jmap_dart_client/http/converter/unsigned_int_nullable_converter.dart';
+import 'package:jmap_dart_client/http/converter/utc_date_nullable_converter.dart';
+import 'package:jmap_dart_client/jmap/mail/email/email_address.dart';
+import 'package:jmap_dart_client/jmap/mail/email/keyword_identifier.dart';
+import 'package:jmap_dart_client/jmap/core/id.dart';
+import 'package:jmap_dart_client/jmap/core/properties/properties.dart';
+import 'package:jmap_dart_client/jmap/core/unsigned_int.dart';
+import 'package:jmap_dart_client/jmap/core/utc_date.dart';
+import 'package:jmap_dart_client/jmap/mail/mailbox/mailbox.dart';
+
+class Email with EquatableMixin {
+  static Properties allProperties = Properties({
+    'id', 'subject','from', 'to', 'cc', 'bcc', 'keywords', 'size', 'receivedAt',
+    'sentAt', 'replyTo', 'preview', 'hasAttachment'
+  });
+
+  final EmailId id;
+  final EmailId? blobId;
```

Comment:
`Id?`

Reply:
When can blboId and threadId be missing???

Reply:
> When can blboId and threadId be missing???

oh. I am mistaken here. It is not a `nullable`. It is shown when we declare it in a request of `email/get` method at `properties`